pkg/odo/cli/cli.go

```diff
@@ -3,7 +3,6 @@ package cli
 import (
 	"flag"
 	"fmt"
-	"github.com/redhat-developer/odo/pkg/odo/util"
```

Comment:
Should we be changing the positioning of imports?

Reply:
That's something I don't understand… it should always be in the same order based on `go fmt` but yet it changes sometimes… :/

Reply:
Same here.. I don't know how this works...

Reply:
Yes I find that really odd as well